lib/features/family/features/profiles/screens/profile_selection_screen.dart

```diff
@@ -86,8 +94,8 @@ class _ProfileSelectionScreenState extends State<ProfileSelectionScreen> {
             isError: true,
           );
         } else if (state is ProfilesNotSetupState) {
-          if (CachedFamilyUtils.isFamilyCacheExist()) {
-            await context.pushNamed(FamilyPages.cachedChildrenOverview.name);
+          if (state.cachedMembers.isNotEmpty && !isCachedMembersBottomsheetUp) {
```

Comment:
I know you asked if there was a better solution for this during the standup. I'll take a look at that outside of this PR.